tests/transforms/test_tape_expand.py

```diff
@@ -624,12 +623,12 @@ def circuit():
             return qml.expval(qml.PauliZ(0))
 
         # BasicEntanglerLayers custom decomposition involves AngleEmbedding. If
-        # expansion depth is 1, the AngleEmbedding will still be decomposed into
+        # expansion depth is 2, the AngleEmbedding will still be decomposed into
         # RX (since it's not a supported operation on the device), but the RX will
         # not be further decomposed even though the custom decomposition is specified.
         custom_decomps = {"BasicEntanglerLayers": custom_basic_entangler_layers, "RX": custom_rx}
         decomp_dev = qml.device(
-            "default.qubit", wires=2, custom_decomps=custom_decomps, decomp_depth=1
+            "default.qubit", wires=2, custom_decomps=custom_decomps, decomp_depth=2
```

Comment:
This fix is needed due to different expansion behaviour in this test (I think)

Reply:
iirc this was actually a bug in `master` - I went down the rabbit hole, and expecting this to work with `depth=1` is incorrect!